frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx

```diff
@@ -0,0 +1,284 @@
+                  disabled={editInfo.tenancy}
+                  type="number"
+                  field={withNameSpace('rentableArea')}
+                  postText="Sq. Ft"
+                />
+              </Row>
+              <Row className="content-item">
+                <Label>Tenancy</Label>
+                <span className="vl"></span>
+                <FastInput
+                  displayErrorTooltips
+                  formikProps={formikProps}
+                  disabled={editInfo.tenancy}
+                  field={withNameSpace('buildingTenancy')}
+                />
+              </Row>
+              <Row className="content-item">
+                <Label>Type of Occupant</Label>
+                <span className="vl"></span>
+                <FastSelect
+                  formikProps={formikProps}
+                  disabled={editInfo.tenancy}
+                  placeholder="Must Select One"
+                  field={withNameSpace('buildingOccupantTypeId')}
+                  type="number"
+                  options={props.occupantTypes}
+                />
+              </Row>
+              <Row className="content-item">
+                <Label>Occupant Name</Label>
+                <span className="vl"></span>
+                <FastInput
+                  displayErrorTooltips
+                  formikProps={formikProps}
+                  disabled={editInfo.tenancy}
+                  field={withNameSpace('occupantName')}
+                />
+              </Row>
+              <Row className="content-item">
+                <Label>Date Lease Expires</Label>
+                <span className="vl"></span>
+                <FastDatePicker
+                  formikProps={formikProps}
+                  disabled={editInfo.tenancy}
+                  field={withNameSpace('leaseExpiry')}
+                />
+              </Row>
+              <Row style={{ justifyContent: 'center' }} className="content-item">
+                <Label>Transfer lease with land?</Label>
+                {/* <span className="vl"></span> */}
+                <Check disabled={editInfo.tenancy} field={withNameSpace('transferLeaseOnSale')} />
+              </Row>
+            </div>
+          </Row>
+          <Row>
+            <div className="valuation">
+              <Row className="valuation-header">
+                <BuildingSvg className="svg" />
+                <h5>Valuation</h5>
+                <FaEdit
+                  size={20}
+                  className="edit"
+                  onClick={() =>
+                    setEditInfo({ ...defaultEditValues, valuation: !editInfo.valuation })
+                  }
+                />
+              </Row>
+              <Row className="val-item">
+                <Label>Net Book Value</Label>
+                <span className="vl"></span>
+                <FastCurrencyInput
+                  formikProps={formikProps}
+                  field="data.buildings.0.financials.0.netbook.value"
```

Comment:
Are we only doing one financial value per building now?

Reply:
Does the review only show the latest financial values?  If so then you may need to determine which ones to show.  Either you will need to show an appropriate year, or show the most recent year that has values.

Reply:
This is what I gathered from the UI form, but will adapt it later today to show the most recent